[IT] Add `Medals` (#8720)
ppy/osu-wiki: wiki/Medals/it.md

Tutte le Medaglie di Introduzione Alle Mod possono essere sbloccate in qualunque [modalità di gioco](/wiki/Game_mode) a meno che non venga specificato diversamente.

## Hush-Hush

Le **medaglie Hush-Hush** sono medaglie "segrete" ottenute compiendo azioni molto specifiche, che non vengono esplicitamente dette al giocatore (da qui proviene il nome). Queste azioni possono variare da completare una beatmap in particolari circostanze a stabilire una giocata con una combinazione di mod inusuale. Nonostante la loro natura segreta, ci sono degli indizi nascosti in giro per aiutare tutti i collezionistsi di medaglie curiosi nella loro ricerca come segue: 

- Il nome della medaglia in sé,
- Le brevi descrizioni delle medaglie,

> - I loro *testi a comparsa*<!-- https://twitter.com/ephemeralis/status/1068054191602638850 --> delle medaglie (se ci sono), visibile passando sopra l'emblema con il cursore.

Reviewer: ```suggestion
- I loro *testi a comparsa*<!-- https://twitter.com/ephemeralis/status/1068054191602638850 --> (se ci sono), visibili passando sopra l'emblema con il cursore.
```

> Siccome le medaglie Hush-Hush sono fatte per essere obiettivi segreti, i requisiti per sbloccarle non sono vengono direttamente detti al pubblico; ad ogni modo, i giocatori possono [collaborare per capire insieme un modo per ottenere tutte le medaglie Hush-Hush qui](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals).

Reviewer: ```suggestion
Siccome le medaglie Hush-Hush sono fatte per essere obiettivi segreti, i requisiti per sbloccarle non sono vengono direttamente rivelati al pubblico; ad ogni modo, i giocatori possono [collaborare per capire insieme un modo per ottenere tutte le medaglie Hush-Hush qui](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals).
```

## Beatmap Spotlight

Le **medaglie Beatmap Spotlight** sono medaglie ottenute completando certi pacchetti beatmap rilasciati come parte dei  [Beatmap Spotlights Mensili](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights). Per sbloccare le medaglie in questa categoria, un giocatore deve giocare tutte le mappe *in qualunque pacchetto delle 4 modalità di gioco* che fanno parte del corrispondente schema di classificazione dei Beatmap Spotlights. È possibile utilizzare le seguenti mods:

- [Mods di riduzione della difficoltà](/wiki/Game_modifier#difficulty-reduction)
- [Mods di aumento della difficoltà](/wiki/Game_modifier#difficulty-increase)
- [xK](/wiki/Game_modifier/xK), [Mirror](/wiki/Game_modifier/Mirror), e [Spun Out](/wiki/Game_modifier/Spun_Out)

Nonostante abbiamo smesso di creare Beatmap Spotlights Mensili dall'aprile del 2018 per dare posto agli [Spotlights Stagionali](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018), tutte le medaglie pertinenti ad esso possono essere ottenute ancora oggi.

## Seasonal Spotlights

> le **medaglie Spotlights Stagionali** sono medaglie ottenute completando determinati pacchetti beatmap rilasciati come parte degli [Spotlights Stagionali](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018). Per sbloccare le medaglie in questa categoria, un giocatore deve giocare tutte le mappe *in qualunque pacchetto delle 4 modalità di gioco* che fanno parte del corrispondente schema di classificazione degli Spotlights Stagionali. È possibile utilizzare le seguenti mods:

Reviewer: ```suggestion
Le **medaglie Spotlights Stagionali** sono medaglie ottenute completando determinati pacchetti beatmap rilasciati come parte degli [Spotlights Stagionali](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018). Per sbloccare le medaglie in questa categoria, un giocatore deve giocare tutte le mappe *in qualunque pacchetto delle 4 modalità di gioco* che fanno parte del corrispondente schema di classificazione degli Spotlights Stagionali. È possibile utilizzare le seguenti mods:
```

- [Mods di riduzione della difficoltà](/wiki/Game_modifier#difficulty-reduction)
- [Mods di aumento della difficoltà](/wiki/Game_modifier#difficulty-increase)
- [xK](/wiki/Game_modifier/xK), [Mirror](/wiki/Game_modifier/Mirror), e [Spun Out](/wiki/Game_modifier/Spun_Out)

Nonostante abbiamo smesso di creare gli Spotlights Stagionali dall'inizio 2020 in favore dell'[attuale sistema di Beatmap Spotlights](/wiki/Beatmap_Spotlights), tutte le medaglie pertinenti ad esso possono essere ottenute ancora oggi.

## Pacchetti Sfida Beatmap

> le **medaglie dei Pacchetti Sfida Beatmap**, o alternativamente conosciute come [*medaglie Pacchetti Abilità*](https://osu.ppy.sh/home/news/2020-11-20-featured-artist-beatmap-updates-from-the-mappers-guild#skill-packs), sono medaglie ottenute al completamento della difficoltà più alta presente in tutte le beatmap presenti in alcuni [Pacchetti Beatmap a Tema](https://osu.ppy.sh/beatmaps/packs?type=theme) o [Pacchetti Beatmap Artista/Album](https://osu.ppy.sh/beatmaps/packs?type=artist) *senza alcuna mod che [riducono la difficoltà](/wiki/Game_modifier#difficulty-reduction) o [speciali](/wiki/Game_modifier#special) abilitate*. Questi pacchetti beatmap sono curati appositamente per mettere alla prova le abilità di ogni giocatore in vari aspetti del gioco riguardanti diversi livelli di abilità, ad esempio all'interno di una particolare gamma di [difficoltà](/wiki/Beatmapping/Star_rating) o con una difficoltà massima.

Reviewer: ```suggestion
Le **medaglie dei Pacchetti Sfida Beatmap**, o altrimenti conosciute come [*medaglie Pacchetti Abilità*](https://osu.ppy.sh/home/news/2020-11-20-featured-artist-beatmap-updates-from-the-mappers-guild#skill-packs), sono medaglie ottenute al completamento della difficoltà più alta presente in tutte le beatmap presenti in alcuni [Pacchetti Beatmap a Tema](https://osu.ppy.sh/beatmaps/packs?type=theme) o [Pacchetti Beatmap Artista/Album](https://osu.ppy.sh/beatmaps/packs?type=artist) *senza alcuna mod che [riduca la difficoltà](/wiki/Game_modifier#difficulty-reduction) o [speciali](/wiki/Game_modifier#special) abilitate*. Questi pacchetti beatmap sono curati appositamente per mettere alla prova le abilità di ogni giocatore in vari aspetti del gioco riguardanti diversi livelli di abilità, ad esempio all'interno di una particolare gamma di [difficoltà](/wiki/Beatmapping/Star_rating) o con una difficoltà massima.
```

> tutte le beatmap nel pacchetto devono essere completate nella modalità di gioco [osu!](/wiki/Game_mode/osu!) per poter ottenere la medaglia, a meno che diversamente specificato.

Reviewer: ```suggestion
Tutte le beatmap nel pacchetto devono essere completate nella modalità di gioco [osu!](/wiki/Game_mode/osu!) per poter ottenere la medaglia, a meno che diversamente specificato.
```
